Source/Core/Core/PowerPC/Jit64/Jit_LoadStorePaired.cpp

```diff
@@ -25,6 +25,9 @@ void Jit64::psq_stXX(UGeckoInstruction inst)
   INSTRUCTION_START
   JITDISABLE(bJITLoadStorePairedOff);
 
+  // For performance, the AsmCommon routines assume address translation is on.
+  FALLBACK_IF(!UReg_MSR(MSR).DR);
```

Comment:
The block cache makes sure MSR.IR and MSR.DR are consistent between when a block is compiled and when it's executed.  (Since we have to check IR anyway, checking DR is basically free.)

@phire asked essentially the same question before, so I probably need to make it more clear somehow.  Maybe it would help if I shoved a bit into JitState instead of accessing MSR directly?